emsdk/tests/test_dyncall.py

```diff
@@ -42,8 +43,10 @@ def test_dyncall(tmpdir):
         subprocess.run(
             [
                 "emcc",
+                "-g4",
                 "-s",
                 "SIDE_MODULE=1",
+                "-sWASM_BIGINT",
```

Comment:
Do we want to keep these flag changes?

Reply:
More [debugging flags](https://emscripten.org/docs/porting/Debugging.html#debug-information) for tests shouldn't hurt. Not too sure about `WASM_BIGINT` but it might be good to build everything with it anyway in the future.

Reply:
Oops, those are left in there, I think bigint shouldn't be in at least until wasm bigint is enabled on everything